CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Use this format please. https://github.com/openebs/linux-utils/blob/master/CODE_OF_CONDUCT.md

Reply:
I just followed the other openebs repo having a similar format, should i change to use as you mentioned above ?

Reply:
yes.

Reply:
we will have to change others as well

Reply:
done